cmd/skopeo/utils.go

```diff
@@ -56,6 +56,7 @@ func sharedImageFlags() ([]cli.Flag, *sharedImageOptions) {
 type dockerImageOptions struct {
 	global         *globalOptions      // May be shared across several imageOptions instances.
 	shared         *sharedImageOptions // May be shared across several imageOptions instances.
+	authFilePath   optionalString      // Path to a */containers/auth.json (not authFilePath is available as shared image option and as non-shared prefixed version).
```

Comment:
I’m not sure what the “not authFilePath is available…” part means. Just guessing, something like “if set,  shared.authFilePath is ignored”, but that does not quite fit.

(One complication in explaining this is that “non-shared” does not necessary imply “prefixed”, consider `skopeo inspect --authfile`, which does not use source/destination prefixes.)

Reply:
updated comment.